src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -0,0 +1,20 @@
+package lotto.domain
+
+enum class Prize(val matchCount: Int, val amount: Int) {
+    THIRD(3, 5000),
+    FOURTH(4, 50000),
+    FIFTH(5, 1500000),
+    SIXTH(6, 2000000000);
```

Comment:
숫자가 너무커서 읽기 힘든경우에는 아래처럼 표기도 가능합니다.
```suggestion
    SIXTH(6, 2_000_000_000);
```